src/app/business/services/instance-list/instance-list.component.ts

```diff
@@ -91,6 +94,35 @@ export class InstanceListComponent implements OnInit {
     console.log("Pausing Instance:", instanceId);
   }
 
+  showWarningDialogFun(instance) {
+    let msg = "<div>Are you sure you want to delete the Instance?</div><h3>[ "+ instance.name +" ]</h3>";
+    let header ="Delete Instance";
+    let acceptLabel = "Delete";
+    let warning = true;
+    this.confirmDialog([msg,header,acceptLabel,warning,instance.id])
+  }
+
+  confirmDialog([msg,header,acceptLabel,warming=true,func]){
+      this.confirmationService.confirm({
+          message: msg,
+          header: header,
+          acceptLabel: acceptLabel,
+          isWarning: warming,
+          accept: ()=>{
+              try {
+                  this.deleteInstance(func);
+              }
+              catch (e) {
+                  console.log(e);
+              }
+              finally {
+                  
```

Comment:
Should we notify here that deleteInstance was failed?

Reply:
This is being handled at line no 135.